packages/edit-site/src/components/page-pages/index.js

```diff
@@ -117,6 +121,18 @@ export default function PagePages() {
 					postStatuses[ page.status ] ?? page.status,
 				enableSorting: false,
 			},
+			{
+				header: 'Date',
+				id: 'date',
+				cell: ( props ) => {
+					const formattedDate = dateI18n(
+						getSettings().formats.datetimeAbbreviated,
+						getDate( props.row.original.date )
```

Comment:
The default edit.php seems to fallback to the modified data for drafts

Reply:
The `date` field in this PR was just a placeholder without the proper handling of this field to showcase the API. We can address this in follow ups.